docs/community/get_involved.md

```diff
@@ -7,28 +9,70 @@
 
 </div>
 
+## GTFS Governance
+
+The GTFS Specification is dynamic, evolving through an open process driven by the community. We actively participate in proposing amendments and extensions to the specification.
```

Comment:
I would suggest that we integrate some the [text from this page](https://gtfs.org/schedule/changes/) (at the top), I think is a good intro to the governance that could complement the text here.

Reply:
I had placed this text as a placeholder. I am not a fan of how we currently describe the process to be completely honest with you.